QualityControl/test/setup/testServerSetup.js

```diff
@@ -34,8 +34,8 @@ export async function setupServerForIntegrationTests() {
   const subprocess = spawn('node', ['index.js', 'test/config.js'], {
     stdio: 'pipe',
     env: {
-      ...process.env,
       NODE_ENV: 'test',
+      ...process.env,
```

Comment:
Any particular reason for this change? As it was written the other way around, it ensured that even of `process.env` would contain the variable `NODE_ENV`, it will not overwrite it. Is it the case that you wish to overwrite the test server setup?